components/portal/main.js

```diff
@@ -434,7 +434,7 @@ define([
         $sessionStorage.portal = $sessionStorage.portal || {};
         configureAppConfig();
 
-        if (APP_FLAGS.loginOnLoad && !lastLoginValid()) {
```

Comment:
Since loginOnLoad was almost always false, we should be able to just drop lines 437-459